docs/data/joy/customization/approaches/ButtonThemes.js

```diff
@@ -560,7 +560,7 @@ export default function ButtonThemes() {
               }}
               size="sm"
               value={design}
-              onChange={setDesign}
+              onChange={(event, newValue) => setDesign(newValue)}
```

Comment:
Why not use?
```suggestion
              onChange={(event) => setDesign(event.target.value)}
```

so developers can switch between an native and non native select?

Reply:
https://github.com/mui/material-ui/pull/34158#issuecomment-1246235105

Reply:
`event.target.value` isn't reliable in this case. The `event` could be a click event on a `li` element that represents a select option.

Reply:
It could be made reliable https://github.com/mui/material-ui/blob/f2d6d61cc4d42b9b2ee9f5ea9ebf8aae192840d4/packages/mui-material/src/Select/SelectInput.js#L285-L295. IMHO, it's outside of the scope of MUI Base, as per definition, I think that you should first use a `<select>`, and then move to a custom component if you need more custom behaviors, but for Joy UI/Material UI, I think that it's in scope.

Reply:
Fair point. I'll create an issue to support `event.target.value` in SelectUnstyled.
As for native selects, I think it could be implemented in Material/Joy, not Base.

Reply:
@michaldudak I think it should not block this PR.

Reply:
Very much agree, it's under the "off-topic" umbrella.https://www.notion.so/mui-org/GitHub-PRs-7112d03a6c4346168090b29a970c0154#fdaee91f0c694d619bb5e1a6586961d0